db/seeds.rb

```diff
@@ -5,3 +5,18 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require "csv"
+
+DATA_DICTIONARY = "#{Rails.root}/lib/data/lat-dict-base.xml"
+doc_dict = File.open(DATA_DICTIONARY) { |f| Nokogiri::XML(f) }
+entries_dict = doc_dict.xpath("//entryFree")
+entries_dict.each do |entry|
+  Dictionary.create(
```

Comment:
What happens if you accidentally ran this 5 times. Check using `find_or_create` or something similar